src/status_im/contexts/wallet/share_community_channel/style.cljs

```diff
@@ -0,0 +1,26 @@
+(ns status-im.contexts.wallet.share-community-channel.style
+  (:require [quo.foundations.colors :as colors]))
+
+(def header-container
+  {:padding-horizontal 20
+   :padding-vertical   12})
+
+(def scan-notice
+  {:color        colors/neutral-40
+   :margin-top   12
+   :margin-left  :auto
+   :margin-right :auto})
+
+(def qr-size 500)
+
+(def qr-code-wrapper
+  {:padding-horizontal 20
+   :align-items        :center})
+
+(defn qr-code-size
+  [total-width]
+  (- total-width (* 2 32)))
+
+(defn gradient-cover-size
+  [total-width]
+  (- total-width (* 2 20)))
```

Comment:
Do we need the multiplication here instead of just 64 and 40? If it's related to some constants, then I think they should be defined as consts to understand what they stand for.